offchain/cli/initiate.ts

```diff
@@ -0,0 +1,109 @@
+import { input, select } from "@inquirer/prompts";
+import { OneshotOneshotMint, ScriptHashRegistry, TreasuryTreasurySpend, VendorVendorSpend } from "../types/contracts";
+import { getMultiSigScript, getProvider, getWallet, transactionDialog } from "./shared";
+import { Blaze, Core, Value } from "@blaze-cardano/sdk";
+import { Address, AssetName, Datum, HexBlob, PolicyId, TransactionId, TransactionInput, TransactionOutput, TransactionUnspentOutput } from "@blaze-cardano/core";
+import { serialize, Void } from "@blaze-cardano/data";
+import { contractsValueToCoreValue } from "../shared";
+
+export async function initiate(): Promise<void> {
+    const bootstrapUtxo = {
+        transaction_id: await input({
+            message: "Enter the transaction ID of the bootstrap UTXO",
+        }),
+        output_index: BigInt(await input({
+            message: "Enter the output index of the bootstrap UTXO",
+        })),
+    };
+
+    const oneshotScript = new OneshotOneshotMint(bootstrapUtxo);
+
+    const registry_token = oneshotScript.Script;
+    console.log(`Registry token policy ID: ${registry_token.hash()}`);
+
+    const treasuryConfig = {
+        registry_token: registry_token.hash(),
+        permissions: {
+            reorganize: await getMultiSigScript("Multisig for treasury reorganize"),
+            sweep: await getMultiSigScript("Multisig for treasury sweep"),
+            fund: await getMultiSigScript("Multisig for treasury fund"),
+            disburse: await getMultiSigScript("Multisig for treasury disburse"),
+        },
+        expiration: BigInt(await input({
+            message: "Enter the expiration time (in seconds since epoch)",
```

Comment:
for me, this message is a little unclear
would be nice to clarify which "epoch" and what the impact of expiration means
see attempt below

```suggestion
            message: "Enter the expiration time for when funds can be swept back to the treasury (in seconds since deployment epoch)",
```

Reply:
The messages are mostly just quick placeholders, so thank you for the input!

Reply:
Understood 🤝